py3status/modules/rss.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+"""
+Display the unread feed items in your OwnCloud/NextCloud account.
+
+Configuration parameters:
+    aggregator: feed aggregator used, see note below (default 'owncloud')
+    cache_timeout: how often to run this check (default 60)
+    feeds_id: list of IDs of feeds to watch, see note below (default [])
+    folders_id: list of IDs of folders ro watch, see note below (default [])
+    format: format to display (default 'Feed: {unseen}')
+    hide_if_zero: don't show on bar if False (default False)
+    new_items_color: text color when counter isn't zero (default '')
+    password: login password (default '<PASSWORD>')
+    server: IMAP server to connect to (default 'https://yourcloudinstance.com')
```

Comment:
As you can see, I cloned another module that I know well in order to make this one :)
Sorry